server/inngest/eventHandlers/sendUpcomingNotifications.ts

```
@@ -0,0 +1,125 @@
+    if (!subscriptionsToNotify.length) return { skipped: true, reason: "No subscriptions need notifications" };
+    
+    const userIds = subscriptionsToNotify.map(s => s.hemocioneId).filter(Boolean) as string[];
+
+    const config = useRuntimeConfig();
+    const hemocioneIdBaseUrl = config.hemocioneIdBaseUrl;
+    const backofficeToken = config.hemocioneIdBackofficeToken;
+    const oneSignalTemplateId = config.oneSignalTemplateId;
+    const whatsappTemplateName = config.whatsappTemplateName;
+
+    if (!backofficeToken || !hemocioneIdBaseUrl) {
+      return { skipped: true, reason: "Missing Hemocione ID credentials/config" };
```

ESLint

[error] Replace `·skipped:·true,·reason:·"Missing·Hemocione·ID·credentials/config"` with `⏎········skipped:·true,⏎········reason:·"Missing·Hemocione·ID·credentials/config",⏎·····`

(prettier/prettier)